lib/ex_binance/rest/cancel_all_orders.ex

```diff
@@ -0,0 +1,35 @@
+defmodule ExBinance.Rest.CancelAllOrders do
+  alias ExBinance.Rest.HTTPClient
+  alias ExBinance.Timestamp
+
+  @type symbol :: String.t()
+  @type credentials :: ExBinance.Credentials.t()
+  @type ok_response :: ExBinance.Responses.CancelOrder.t()
+  @type error_msg :: String.t()
+  @type error_reason :: {:not_found, error_msg} | ExBinance.Rest.HTTPClient.shared_errors()
+
+  @path "/api/v3/openOrders"
+  @receiving_window 1000
+
+  @spec cancel_all_orders(symbol, credentials) ::
```

Comment:
Can you add a test for this? e.g. https://github.com/fremantle-capital/ex_binance/blob/master/test/ex_binance/private/cancel_order_test.exs

Reply:
Will do.

Reply:
https://github.com/fremantle-capital/ex_binance/blob/dfb0c75385a7df6316f4788a766e147d495ae00a/test/ex_binance/private/cancel_all_orders_test.exs